_pages/model-deployment/nltk.md

```diff
@@ -8,6 +8,7 @@ show_related: true
 author: steph_kim
 permalink: /model-deployment/nltk/
 redirect_from:
+  - /algorithm-development/guides/nltk-guide
```

Comment:
The Google Doc says the new URL for this one should be `/clients`, but your approach seems like a better user experience as it's more in line with what the user was trying to find. Can you confirm with Mel/Whitney that this is okay, if you haven't already?

Also, let's add a trailing slash to this value.

Reply:
@wnistrian